tests/topotests/ospf_basic_functionality/test_ospf_asbr_summary_type7_lsa.py

```diff
@@ -233,7 +233,7 @@ def red_connected(dut, config=True):
         ospf_red = {
             dut: {
                 "ospf": {
-                    "redistribute": [{"redist_type": "connected", "del_action": True}]
+                    "redistribute": [{"redist_type": "connected", "delete": True}]
```

Comment:
If you look below, the function being called is `create_router_ospf`. This function did not handle the old argument of `del_action`, so this change means this config is now being removed (which one expects) whereas before it was not, yet the tests passed previously.

Also note `create_rotuer_ospf` calls `__create_ospf_global` which then has multiple custom keys besides `delete` for various config which it should not; however, this does not include `del_action` as mentioned above.

I didn't check each of these changes, but you should go back and do so.